### PR TITLE
added swagger docs for application endpoints

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chaterjee <speak2adi@gmail.com>

**Description**
Added swagger docs for the /application endpoints.
This PR fixes #

**Notes for Reviewers**
Sorry, this took so long

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: handlers/doc.go [hunk 1]
```diff
@@ -316,3 +316,24 @@ type applicationFileParamsWrapper struct {
...
+	Body models.MesheryApplication
+}
+
+// Posts Application file requests
+// swagger:response mesheryApplicationFileRequestWrapper
+type mesheryApplicationFileRequestWrapper struct {
+	//in: body
+	Body models.MesheryApplication
+}
+
```
Comment (piyushsingariya) [inline handlers/doc.go:333]: two same responses?

File: handlers/meshery_application_handler.go [hunk 2]
```diff
@@ -44,6 +44,20 @@ func (h *Handler) ApplicationFileHandler(
 	h.PatternFileHandler(rw, r, prefObj, user, provider)
 }
 
+// swagger:route POST /api/experimental/application/ ApplicationsAPI idPostApplicationFileRequest
+// Handle POST request for Application Files
+//
+// Deploys requests for all Meshery Application Files
```
Comment (piyushsingariya) [inline handlers/meshery_application_handler.go:50]:
```suggestion
// Save attached Meshery Application File
```

File: handlers/doc.go (hunk 1 above)
Comment (piyushsingariya) [reply handlers/doc.go:333]: Merge them!

**Review by piyushsingariya — CHANGES_REQUESTED** [review]
run `make swagger-spec` and `make swagger-docs` and include those changed too with the PR.

**Comment by l5io** [comment]
🚀 Preview for commit 8869586b73d614d2c45607e80485ff7b9eb5ed83 at: https://611248db044c783c6aff5f9b--meshery-docs-preview.netlify.app

**Comment by warunicorn19** [comment]
@piyushsingariya because of the error while running `make swagger-docs`, This swagger.yml file is getting deleted every time this command in executed.

**Comment by piyushsingariya** [comment]
Let's get on a call and resolve the issue with the error!

**Comment by warunicorn19** [comment]
Yes @piyushsingariya  let's do that, maybe some time tomorrow, I have my placement classes starting from 7. // @sayantan1413

**Pull request closed without merging**